projects/plugins/jetpack/package.json

```diff
@@ -58,6 +58,7 @@
 		"@automattic/popup-monitor": "1.0.2",
 		"@automattic/request-external-access": "1.0.0",
 		"@automattic/social-previews": "2.0.1-beta.10",
+		"@automattic/tour-kit": "1.1.1",
```

Comment:
This is the package being used to make the Welcome tour. You can see an example of the usage here: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Scyhtvaf%2Srqvgvat%2Qgbbyxvg%2Qcyhtva%2Scebq%2Sjcpbz%2Qoybpx%2Qrqvgbe%2Qahk%2Sfep%2Sjrypbzr%2Qgbhe%2Sgbhe%2Qynhapu.gfk%3Se%3Qpsr9p3so%234-og